src/modules/features/SettingDialogue/ObsidianLiveSyncSettingTab.ts

```diff
@@ -1526,8 +1533,8 @@ However, your report is needed to stabilise this. I appreciate you for your grea
 
             void addPanel(paneEl, "Fetch settings").then((paneEl) => {
                 new Setting(paneEl)
-                    .setName("Fetch tweaks from the remote")
-                    .setDesc("Fetch other necessary settings from already configured remote.")
+                    .setName("Fetch config from server")
```

Comment:
Please keep `remote` or change something good: we can serve some database (or bucket) on a single server. (It used to be Database, before implemented bucket sync) However, as you fixed, this is a bit ambiguous.